page-components/ArtistDrive/SongThumbnail.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { rem } from 'polished';
+import styled from 'styled-components';
+import { Label3, Label4 } from 'components/Typography';
+import { Button } from 'components/Button';
+import type { SongThumbnailProps } from './types';
+import { formatSongLength } from 'utilities/timeUtils';
+
+
+const SongThumbnail: React.FC<SongThumbnailProps> = (props: SongThumbnailProps) => {
```

Comment:
We normally just define the props types right above the component in the same file